app/controllers/enrollments_controller.rb

```
@@ -82,6 +83,31 @@ def show
       @position_is_set = true
     end
 
+
+    if @enrollment.campaign_id
+      sf = BeyondZ::Salesforce.new
+      client = sf.get_client
+      client.materialize('Campaign')
+      campaign = SFDC_Models::Campaign.find(@enrollment.campaign_id)
+
+      # It might be worth caching this at some point too.
+
+      if campaign
+        @program_title = campaign.Program_Title__c
+        @program_site = campaign.Program_Site__c
+        @request_availability = campaign.Request_Availability__c
+        @meeting_times = campaign.Meeting_Times__c
+      end
+    end
+
+    # HACK FOR TESTING WITHOUT SALESFORCE
```

Hmm.  For production, this should throw an exception or have reasonable defaults and log it on the backend that we used those.